src/systems/ec/implementations/react/components/radio/src/RadioGroup.jsx

```diff
@@ -1,92 +1,119 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 import RadioButton from './RadioButton';
 
 const RadioGroup = ({
-  labelId,
-  items,
-  name,
-  helperText,
-  label,
+  binary,
   helperId,
-  hideLabel,
+  helperText,
   invalid,
   invalidText,
-  binary,
+  items,
+  label, // DEPRECATED
+  labelId, // DEPRECATED
+  legend,
+  legendClassName,
+  legendId,
+  name,
+  optionalText,
+  required,
+  requiredText,
   className,
   ...props
 }) => {
-  const classNames = classnames(className, 'ecl-radio__group', {
-    'ecl-radio__group--invalid': invalid,
-    'ecl-radio__group--binary': binary,
-  });
+  const classNames = classnames(
+    className,
+    'ecl-form-group ecl-form-group--radio',
+    {
+      'ecl-radio__group--binary': binary,
+    }
+  );
 
   return (
-    <div
+    <fieldset
       {...props}
-      role="radiogroup"
-      aria-labelledby={labelId}
+      aria-labelledby={labelId || legendId}
```

Comment:
now that you're using `fieldset` and `legend`, you don't need `aria-labelledby` anymore